trick_source/sim_services/CommandLineArguments/CommandLineArguments.cpp

```diff
@@ -163,6 +163,11 @@ int Trick::CommandLineArguments::process_sim_args(int nargs , char **args) {
         } else {
             run_dir = "." ;
         }
+        /* check existence of run directory */
+        if (access(run_dir.c_str(), F_OK) != 0) {
+            std::cerr << "\nERROR: run directory \"" << run_dir << "\" does not exist. Exiting!" << std::endl ;
```

Comment:
`access` can fail for a number of reasons besides the file not existing. Use [`std::strerror(errno)`](http://en.cppreference.com/w/cpp/string/byte/strerror) to let the system tell the user what the problem was.